include/aws/common/process.h

```diff
@@ -21,6 +21,8 @@ AWS_EXTERN_C_BEGIN
 
 AWS_COMMON_API int aws_get_pid(void);
 
+AWS_COMMON_API struct aws_string *aws_get_command_output(struct aws_allocator *allocator, const char *command);
```

Comment:
wouldn't we also want the exit code?

Reply:
How does this deal with stderr vs stdout? Do we get both? stdout only?

Reply:
Maybe we should just rip off python with all of our hearts: https://docs.python.org/3/library/subprocess.html#subprocess.run
and make something like:
```
struct aws_process_completed *aws_process_run(struct aws_allocator *allocator, const struct aws_process_run_options *options);
```

`options` could be something that gets more complex as time goes on and we have more diverse requirements
`aws_process_completed` could start with just having `return_code` and `stdout` in it, but we could add things like `stderr` in the future when we need it

Reply:
popen only gives you stdout , no stderr.
I'll take a look at the python definition, thanks.